frontend/src/utilities/__tests__/clusterUtils.spec.ts

```diff
@@ -0,0 +1,117 @@
+import * as clusterInfo from '~/redux/selectors/clusterInfo';
+import { getOpenShiftConsoleServerURL, useOpenShiftURL } from '~/utilities/clusterUtils';
+import { testHook } from '~/__tests__/unit/testUtils/hooks';
+let originalLocation: Location;
+
+const setupWindowLocation = (hostname: string, protocol: string, port: string) => {
+  Object.defineProperty(window, 'location', {
+    writable: true,
+    value: { ...originalLocation, hostname, protocol, port },
+  });
+};
+
+jest.mock('~/utilities/const', () => ({
+  get DEV_MODE() {
+    return false;
+  },
+}));
+
+jest.mock('~/redux/selectors/clusterInfo', () => ({
+  ...jest.requireActual('~/redux/selectors/clusterInfo'), // Use the actual module for other exports
+  useClusterInfo: jest.fn(),
+}));
```

Comment:
Use this method of mocking or `spyOn` but not both. Prefer the use of this mocking and only using `spyOn` for special circumstances.